test_hunyuan_dit_lora.py

```diff
@@ -0,0 +1,13 @@
+import torch
```

Comment:
we will remove this file before merge, no?
in the future, maybe it's easier to just post testing example in PR description:)

Reply:
Yes. Additionally, I think this should be turned into a proper test suite: `test_lora_layers_hunyuan_dit.py`. Just including a SLOW test is sufficient for the time being. 

Here is an example: https://github.com/huggingface/diffusers/blob/d457beed92e768af6090238962a93c4cf4792e8f/tests/lora/test_lora_layers_sd.py#L205.